code_examples/sdk_examples/src/dapp/verifier/verifyCredentialMessage.ts

```diff
@@ -52,7 +52,7 @@ export async function main({ session, verifierKeys }: Param) {
     try {
       await Kilt.Credential.verifyPresentation(credential)
     } catch (e) {
-      console.log('credential was invalid:', e)
+      console.log('Credential was invalid:', e)
       return
```

Comment:
It's not your code but maybe we should throw here.

Reply:
https://github.com/KILTprotocol/docs/pull/201/commits/d63cff7b0b3a61dfb01914c3262e3ff854216082